### ISSUE TITLE
Marking embedded/transcluded content as remote (e.g. add h-cite class)

### ISSUE THREAD

**Issue by elf-pavlik** [converspace/webmention#40]
After [Social WG F2F3](https://www.w3.org/wiki/Socialwg/2015-05-04) we had conversation with @rhiaro about [Named Graphs](http://www.w3.org/TR/rdf11-primer/#section-multiple-graphs) and keeping provenance of data on decentralized social web. We also discussed how we understand IndieWeb community use of [h-cite microformat](http://microformats.org/wiki/h-cite)

We concluded with an idea to discuss recommending making all the content added via webmention as **remote**. In case of sites using microformats, it could work by simply adding additional class *h-cite* eg. class="h-entry h-cite", class="h-card h-cite", class="h-event h-cite" etc. @tantek

It could go under [Preventing Abuse](https://github.com/converspace/webmention/blob/master/README.md#preventing-abuse) section of this spec.

**Comment by aaronpk** [comment]
I agree with this, it seems equivalent to rel="nofollow". However I think things that talk about the display of comments shouldn't be part of the core Webmention spec, so maybe just referencing this non-normatively.

**Comment by elf-pavlik** [comment]
Yes, I think webmention should NOT mandate h-cite or couple with microformats. Non-normative note similar to rel="nofollow" sounds very reasonable, especially that as of today very likely most people may use it together with microformats.
